Automattic/wp-calypso
# Pattern Assembler: Update styles on the main screen

#### Proposed Changes

* Update the copy of the description
* Update the styles of the main screen

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/203491093-a708c58b-c4d9-4334-896b-073eee51044b.png) | ![image](https://user-images.githubusercontent.com/13596067/203491618-f8ada2d4-30fb-43c5-90c0-2f8f3abb5e1c.png) |
| ![image](https://user-images.githubusercontent.com/13596067/203491144-d9f9e331-9823-4791-a285-1163e3acaed2.png) | ![image](https://user-images.githubusercontent.com/13596067/203688338-090e9004-d3ee-40ea-8d19-574e849f83b3.png) |

Hover styles like any other link. 

![image](https://user-images.githubusercontent.com/13596067/203492200-094bad60-8aa5-4469-8a46-94a9405834c9.png)

#### Testing Instructions



* Go to`/setup/site-setup?siteSlug=<your_site>`
* Continue until you land on the Design Picker
* Click the "start designing" button at the end of the Design Picker
* Verify whether the new styles look good

#### Pre-merge Checklist





Related to https://github.com/Automattic/dotcom-forge/issues/1251
